src/BeamAdapter/component/forcefield/AdaptiveBeamForceFieldAndMass.inl

```diff
@@ -689,6 +691,8 @@ template<class DataTypes>
 void AdaptiveBeamForceFieldAndMass<DataTypes>::addDForce(const MechanicalParams* mparams,
                                                          DataVecDeriv& datadF, const DataVecDeriv& datadX )
 {
+    ScopedAdvancedTimer timer("AdaptiveBeamForceFieldAndMass_addDForce");

```

Comment:
I am not sure that this timer is relevant. It will be repeated many times depending on the number of iterations of the conjugate gradient.